### PR TITLE
SAMBRO: start with alerting authority model

### PR DESCRIPTION
**Pull request by biplovbhandari**
@flavour Please merge

### CODE REVIEW

File: modules/s3db/cap.py [hunk 1]
```diff
@@ -3046,6 +3049,148 @@ def list_string_represent(string, fmt=lambda v: v):
...
+                                                      "joinby": "alerting_authority_id",
+                                                      },
+                            cap_authority_forecast_url = {"name": "forecast_url",
+                                                          "joinby": "alerting_authority_id",
+                                                          },
+                            )
+
+        # ---------------------------------------------------------------------
+        # Feed URL for CAP Alerting Authority
+        tablename = "cap_authority_feed_url"
```
Comment (flavour) [inline modules/s3db/cap.py]: Are there multiple of these per Authority?

File: modules/s3db/cap.py [hunk 2]
```diff
@@ -3046,6 +3049,148 @@ def list_string_represent(string, fmt=lambda v: v):
...
+                     # @ToDo: Add username and pwd for url
+                     Field("url",
+                           label = T("CAP Feed URL"),
+                           requires = IS_EMPTY_OR(IS_URL()),
+                           ),
+                     *s3_meta_fields())
+
+        # ---------------------------------------------------------------------
+        # Forecasts URL for CAP Alerting Authority
+        tablename = "cap_authority_forecast_url"
```
Comment (flavour) [inline modules/s3db/cap.py]: Are there multiple of these per Authority?

File: modules/templates/SAMBRO/cap_alerting_authority.csv [hunk 3]
```diff
@@ -0,0 +1,264 @@
+OID,Organisation,Country,Feed URL,Forecast URL,Authorization
+2.49.0.0.4.0,Afghan Meteorological Authority,AF,,http://www.worldweather.org/115/m115.htm,
+2.49.0.0.8.0,Hydrometeorological Institute,AL,,,
+2.49.0.0.12.0,Office National de la M�t�orologie,DZ,,http://www.worldweather.org/122/m122.htm,
+2.49.0.0.24.0,Instituto Nacional de Hidrometeorolog�a e Geof�sica,AO,,http://www.worldweather.org/129/c00261.htm,
```
Comment (flavour) [inline modules/templates/SAMBRO/cap_alerting_authority.csv]: UTF-8 not ANSI

**Comment by biplovbhandari** [comment]
@flavour Please merge

**Pull request merged**